.devcontainer/docker-compose.yml

```diff
@@ -13,30 +13,3 @@ services:
     # Overrides default command so things don't shut down after the process ends.
     command: sleep infinity
 
-    # Runs app on the same network as the database container, allows "forwardPorts" in devcontainer.json function.
-    network_mode: service:db
-
-    # Uncomment the next line to use a non-root user for all processes.
-    # user: vscode
-
-    # Use "forwardPorts" in **devcontainer.json** to forward an app port locally.
-    # (Adding the "ports" property to this file will not forward from a Codespace.)
-
-  db:
```

Comment:
Just seemed like this was overkill since we're often using dev environments in edm-data now. Figure it makes more sense to set up postgres locally as-needed and default to no db, free up some resources when developing locally

Reply:
Maybe I'll add these back in "docker-compose_db.yml" or something like that so the option is easily there